src/main/java/core/basesyntax/figure/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figure;
+
+public class Rectangle extends AbstractFigure implements Figure {
```

Comment:
The Rectangle class is implementing the Figure interface while also being defined as a class. This is redundant since it's already extending AbstractFigure which should implement Figure.